<?php

namespace Container8uxdFzx;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getApp_Auth_ControllerService extends App_KernelDevDebugContainer
{
    /**
     * Gets the public 'app.auth.controller' shared autowired service.
     *
     * @return \App\Controller\AuthController
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/framework-bundle/Controller/AbstractController.php';
        include_once \dirname(__DIR__, 4).'/src/Controller/ApiController.php';
        include_once \dirname(__DIR__, 4).'/src/Controller/AuthController.php';

        $container->services['app.auth.controller'] = $instance = new \App\Controller\AuthController(new \App\Repository\UsersRepository('Symfony\\Component\\Security\\Core\\Encoder\\UserPasswordEncoderInterface'), 'Psr\\Log\\LoggerInterface');

        $instance->setContainer(($container->privates['.service_locator.5nX7ca3'] ?? $container->load('get_ServiceLocator_5nX7ca3Service'))->withContext('app.auth.controller', $container));

        return $instance;
    }
}
